cmd/sealer/cmd/alpha/umount.go

```diff
@@ -0,0 +1,168 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/sealerio/sealer/common"
+	imagecommon "github.com/sealerio/sealer/pkg/define/options"
+	"github.com/sealerio/sealer/pkg/imageengine"
+	"github.com/sealerio/sealer/pkg/imageengine/buildah"
+
+	"github.com/containers/storage"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var umountAll bool
+
+var longUmountCmdDescription = `
+umount the cluster image and delete the mount directory
+`
+
+var exampleForUmountCmd = `
+  sealer alpha umount my-image
+  sealer alpha umount ba15e47f5969
+`
+
+func NewUmountCmd() *cobra.Command {
+	umountCmd := &cobra.Command{
+		Use:     "umount",
+		Short:   "umount cluster image",
+		Long:    longUmountCmdDescription,
+		Example: exampleForUmountCmd,
+		Args:    cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var cid string
+			var imgName string
+
```

Comment:
Please pre-check the args here. Otherwise, errors may be reported in the subsequent flow.

![image](https://user-images.githubusercontent.com/31209634/212006340-fdf52508-3c40-4a4d-842e-5a8cdba533f7.png)